src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -726,6 +726,36 @@ RequestContext<object> requestContext
                     this.editorSession,
                     eventContext);
             }
+
+            // Convert the editor file glob patterns into an array for the Workspace
+            var excludeFiles = new List<string>();
+            if (configChangeParams.Settings.Files?.Exclude != null)
+            {
+                foreach(string pattern in configChangeParams.Settings.Files.Exclude.Keys)
```

Comment:
Saves having to rehash the file glob to lookup the bool.

```suggestion
                foreach(KeyValuePair<string, bool> excludedFile in configChangeParams.Settings.Files.Exclude)
```

Reply:
Done